translations/nl/news.po

```diff
@@ -4,6 +4,8 @@ msgstr ""
 
 msgid "article .index"
 msgstr ""
+"report_IPv6-meeting\n"
```

Comment:
As per the README on the content repo:
>Note: this translation between msgids and filenames imposes a limit on the valid characters for msgids. It is advised to limit msgids to alphanumeric, space (for seperation) and dash('-').

Having an underscore on msgids will create problems with the conversion script IIRC.